File: backoffice/src/app/services/data/Produit.ts
import {EtatProduit} from '../Enum/EtatProduit';

export class Produit{
  id: number;
  details: string;
  etat: EtatProduit;
  identifiant: string;
  montant: number;
  nom: string;
  quantite: number;
}

File: donne/src/main/java/freelance/paiement/donne/models/Produit.java
package freelance.paiement.donne.models;

import Enum.EtatProduit;
import lombok.*;

import javax.persistence.*;
import javax.validation.constraints.NotNull;

@Entity
@Getter
@Setter
public class Produit{
    @Id
    @GeneratedValue
    private Long id;
    private String details;
    @Enumerated
    private EtatProduit etat;
    private String identifiant;
    @NotNull
    private Float montant;
    private String nom;
    private Integer quantite;
}
